### ISSUE TITLE
Use internal derefer

### ISSUE THREAD

**Issue by CodeInsider** [satoruk/redmine_external_link#2]
Hi there,

it would be better to have a internal derefer as default instead of one in the internet, which destroys in my eyes the use-case of this plugin.

I could provide you with a patch, which shows how we used to solve this.

**Comment by jpschw** [comment]
I agree that this and any plugin should not have an external dependency in order to function unless absolute necessary.
